pkg/api/types.go

```diff
@@ -283,6 +283,8 @@ type LivenessProbe struct {
 	Exec *ExecAction `json:"exec,omitempty"`
 	// Length of time before health checking is activated.  In seconds.
 	InitialDelaySeconds int64 `json:"initialDelaySeconds,omitempty"`
+	// Length of time before health checking times out.  In seconds.
+	TimeoutSecs int64 `json:"timeoutSecs,omitempty"`
```

Comment:
Will we ever want sub second timeouts?  I can think of some health checks that 100ms is considered failure - since it's an int64, why not make this Timeout in milliseconds (or use resource.Quantity)?